### PR TITLE
[IMP] Add partner to filter

### PR DESCRIPTION
**Pull request by cubells**
Allows user to search for a particular partner in invoice payments.

### CODE REVIEW

File: account_banking_payment_export/wizard/payment_order_create.py [hunk 1]
```diff
@@ -31,6 +31,7 @@ class PaymentOrderCreate(models.TransientModel):
 
     journal_ids = fields.Many2many(
         'account.journal', string='Journals Filter')
+    partner_id = fields.Many2one(comodel_name='res.partner', string='Partner')
```
Comment (pedrobaeza) [inline account_banking_payment_export/wizard/payment_order_create.py]: Better to be a many2many to allow multiple partners to filter

**Comment by cubells** [comment]
Done!

File: account_banking_payment_export/wizard/payment_order_create.py [hunk 2]
```diff
@@ -125,6 +127,9 @@ def search_entries(self):
                   ('reconcile_id', '=', False),
                   ('company_id', '=', payment.mode.company_id.id),
                   ('journal_id', 'in', journals.ids)]
+        if self.partner_ids:
+            domain.append(
+                ('partner_id', 'in', [x.id for x in self.partner_ids]))
```
Comment (pedrobaeza) [inline account_banking_payment_export/wizard/payment_order_create.py]: Use self.partner_ids.ids instead

**Comment by pedrobaeza** [comment]
:+1:

**Comment by elicoidal** [comment]
:+1:

**Comment by sandrafig** [comment]
👍

**Pull request merged**